metriq_gym/cli.py

```diff
@@ -12,6 +12,11 @@ def parse_arguments() -> argparse.Namespace:
         Parsed arguments as an argparse.Namespace object.
     """
     parser = argparse.ArgumentParser(description="Quantum volume certification")
+    parser.add_argument(
+        "input_file",
```

Comment:
Do we want to add `required = True` as well for this one? I can't think of a case where we wouldn't want to run something without a schema.

Reply:
That's true by default for positional arguments.